### DISCUSSION TITLE
Any plans for GraphQL API?

### DISCUSSION THREAD

**Discussion by amxv** [Thinkmill/keystatic#979] [Q&A] [answered]
first off, @simonswiss and team - great job on building keystatic! i'm new here and keystatic looks like exactly what I was looking for to manage content in an Astro SSG site. I admire the simplicity in both the DX and UX for non-technical content managers.

Just curious if you have any plans to build a graphql api / db integration in the future like Tina CMS?

**Comment by simonswiss** [comment, marked as answer]
Hey!

As of now there are no concrete plans for this. Worth mentioning, what you're describing (having a DB + API) sounds a lot like Keystatic's older cousin [Keystone](https://keystonejs.com), built by the same team. 

For more ambitious and CRUD-capable projects, you might want to look into that as well!
